### PR TITLE
Update storage xplat-cli guide doc

### PR DESCRIPTION
**Pull request by slepox**

### CODE REVIEW

File: articles/storage-xplat-guide-full.md [hunk 1]
```diff
@@ -111,76 +110,88 @@ While most of the storage commands will work without an Azure subscription, we r
...
+Then copy the output connection string and set it to environment variable:
+
+        export AZURE_STORAGE_CONNECTION_STRING=<connection_string>
 
 ## How to manage Azure blobs
 Azure Blob storage is a service for storing large amounts of unstructured data, such as text or binary data, that can be accessed from anywhere in the world via HTTP or HTTPS. This section assumes that you are already familiar with the Azure Blob Storage Service concepts. For detailed information, see [How to use Blob Storage from .NET](storage-dotnet-how-to-use-blobs.md) and [Blob Service Concepts](http://msdn.microsoft.com/library/azure/dd179376.aspx).
 
 ### How to create a container
-Every blob in Azure storage must be in a container. You can create a private container using the **azure storage container create** cmdlet:
+Every blob in Azure storage must be in a container. You can create a private container using the New-AzureStorageContainer command:
```
Comment (yaxia) [inline articles/storage-xplat-guide-full.md]: Why do you change it back to PSH cmdlet?

File: articles/storage-xplat-guide-full.md [hunk 2]
```diff
@@ -19,15 +19,14 @@
 
 ## Overview
 
-In this guide, we’ll explore how to use [Azure Cross-Platform Command-Line Interface (xplat-cli)](http://azure.microsoft.com/en-us/documentation/articles/xplat-cli/) to perform a variety of development and administration tasks with Azure Storage. 
+In this guide, we’ll explore how to use [Azure Cross-Platform Command-Line Interface (xplat-cli)](http://azure.microsoft.com/en-us/documentation/articles/xplat-cli/) (xplat-cli in short) to perform a variety of development and administration tasks with Azure Storage.
 
-Xplat-cli provides a set of open source, cross-platform commands for working with the Azure Platform. The xplat-cli provides much of the same functionality found in the Azure Management Portal. 
+Azure xplat-cli provides a set of open source, cross-platform commands for working with the Azure Platform. It provides much of the same functionality found in the Azure Management Portal as well as rich functionalities of data access.
```
Comment (yaxia) [inline articles/storage-xplat-guide-full.md]: May they be both "functionality" or "functionalities"?

Comment (slepox) [inline articles/storage-xplat-guide-full.md]: yes, should be the same.

File: articles/storage-xplat-guide-full.md [hunk 3]
```diff
@@ -111,79 +112,91 @@ While most of the storage commands will work without an Azure subscription, we r
 ### How to create a new Azure storage account
 To use Azure storage, you will need a storage account. You can create a new Azure storage account after you have configured your computer to connect to your subscription. 
 
-        azure storage account create
+        azure storage account create <account_name>
 
 > [AZURE.IMPORTANT] The name of your storage account must be unique within Azure and must be lowercase. For naming conventions and restrictions, see [About Azure Storage Accounts](storage-create-storage-account.md) and [Naming and Referencing Containers, Blobs, and Metadata](http://msdn.microsoft.com/library/azure/dd135715.aspx).
```
Comment (yaxia) [inline articles/storage-xplat-guide-full.md]: The link: http://msdn.microsoft.com/library/azure/dd135715.aspx is not about naming of storage accounts. 
 The statement of storage accounts name I can find is: "A name for the storage account that is unique within Azure. Storage account names must be between 3 and 24 characters in length and use numbers and lower-case letters only." (https://msdn.microsoft.com/en-us/library/azure/hh264518.aspx)

Comment (slepox) [inline articles/storage-xplat-guide-full.md]: No, that is about blob naming. The REST method page seems better

**Pull request merged**